frontend/src/app/utils/testResults.ts

```diff
@@ -91,6 +91,7 @@ export const displayGuidance = (results: MultiplexResults) => {
     hasDiseaseSpecificResults(results, MULTIPLEX_DISEASES.COVID_19) ||
     hasPositiveFluResults(results) ||
     hasPositiveRsvResults(results) ||
-    hasDiseaseSpecificResults(results, MULTIPLEX_DISEASES.HIV)
+    hasDiseaseSpecificResults(results, MULTIPLEX_DISEASES.HIV) ||
+    hasDiseaseSpecificResults(results, MULTIPLEX_DISEASES.SYPHILIS)
```

Comment:
Does this change need to be included with this PR? I don't think this is affecting the org admin level dashboard right?